### PR TITLE
fix: add back waku discv5 metrics

### PR DESCRIPTION
**Pull request by SionoiS**
I noticed that waku discv5 metrics were not used so I added them back.

### CODE REVIEW

File: waku/discovery/waku_discv5.nim [hunk 1]
```diff
@@ -211,6 +211,8 @@ proc findRandomPeers*(
   elif wd.predicate.isSome():
     discoveredRecords = discoveredRecords.filter(wd.predicate.get())
 
+  waku_discv5_discovered.inc(discoveredRecords.len)
```
Comment (gabrielmer) [inline waku/discovery/waku_discv5.nim:214]: The only concern that I have is if we receive the same peer twice, we will update this metric both times.

Another alternative is increasing it in `addPeer()` if the origin is `PeerOrigin.Discv5` after passing the duplicate checks

Comment (SionoiS) [reply waku/discovery/waku_discv5.nim:214]: :thinking:  I thought we had another metric for that...

Comment (gabrielmer) [reply waku/discovery/waku_discv5.nim:214]: Sounds good :) Thank you!

**Review by gabrielmer — APPROVED** [review]
LGTM, thanks so much!

File: waku/discovery/waku_discv5.nim [hunk 2]
```diff
@@ -233,6 +235,7 @@ proc searchLoop(wd: WakuDiscoveryV5) {.async.} =
       let peerInfo = record.toRemotePeerInfo().valueOr:
         ## in case of error, we keep track of it for debugging purposes
         wrongRecordsReasons.add(($record, $error))
+        waku_discv5_errors.inc()
```
Comment (Ivansete-status) [inline waku/discovery/waku_discv5.nim:238]: That's nice! Maybe we could also add error detail
```suggestion
        waku_discv5_errors.inc(labelValues = [$error])
```

**Review by Ivansete-status — APPROVED** [review]
LGTM! Thanks for it :100: !
I just added a minor suggestion

**Pull request merged**